flocker/control/test/test_httpapi.py

```diff
@@ -3048,7 +3048,6 @@ def test_one_container(self):
         ])
         expected_dict = dict(
             name=u"myapp",
-            host=expected_hostname,
```

Comment:
Yay for minuses and stuff.  Booo for so much duplication in our test suite. :cry: